scripts/e2e_setup_cluster.sh

```diff
@@ -153,7 +153,7 @@ retry kubectl -n kube-system wait --for=condition=ready -l name="${CNIS_NAME}" p
 echo "## install NRI"
 retry kubectl create -f "${root}/deployments/auth.yaml"
 # enable HugePage API at webhook
-awk '/-logtostderr/ { print; print "    - -injectHugepageDownApi"; next }1' "${root}/deployments/server.yaml" > "${root}/deployments/server_huge.yaml"
+awk '/-logtostderr/ { print; print "        - -injectHugepageDownApi"; next }1' "${root}/deployments/server.yaml" > "${root}/deployments/server_huge.yaml"
```

Comment:
nit: accidental spaces ?

Reply:
nope the lane was failing because of bad indentation so I fix it